src/main/webapp/filterScript.js

```diff
@@ -0,0 +1,43 @@
+const UNCLICKED = 'unclicked-filter';
+const CLICKED = 'clicked-filter';
+const CLICKED_ARRAY_URL = 'clicked-array=';
+
+/**
+* Turn a filter a different color when pressed, and change class accordingly.
```

Comment:
Because the class change is what makes the filter color change, I think that should be the emphasis. Something like "swap between setting the clicked and unclicked classes (which will change the color of the button) on a button when it is clicked."